src/target/riscv/batch.h

```diff
@@ -55,8 +55,10 @@ void riscv_batch_free(struct riscv_batch *batch);
 /* Checks to see if this batch is full. */
 bool riscv_batch_full(struct riscv_batch *batch);
 
-/* Executes this scan batch. */
-int riscv_batch_run(struct riscv_batch *batch);
+/* Executes this scan batch. If resets_delays is true, after
+ * reset_delays_after_n_scans does not insert RTI cycles between scans. */
```

Comment:
I have tried to make the comment more clear:

```suggestion
/* Executes this batch of JTAG DTM DMI scans. 
 *
 * If resets_delays is true, the algorithm will stop inserting idle cycles (JTAG Run-Test Idle)
 * after "reset_delays_after" number of scans is performed. This is useful for testing of algorithms
 * in the OpenOCD's RISC-V debug implementation that are based on batches.
 */
```

Reply:
Thanks! Addressed.